MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -96,13 +98,12 @@ def fetch_new_issues(repo, since, auth_token, db):
 
 		for r in response:
 			if 'pull_request' not in r:
-				store_issue_details(r, repo, db)
-				num_open_issues += 1
+				store_issue_details(r, repo, db)	
```

Comment:
Could you remove the extra tab/space here?

Reply:
Can you please check if it is fine now? Thanks!